<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalisisGrafico</class>
 <widget class="QDialog" name="AnalisisGrafico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analisi Grafico Angulos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonRescalar">
     <property name="styleSheet">
      <string notr="true">QPushButton{
color : white;
background: #0994f0;
border: 2px solid #0e6699;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background: #1077b3;
border: 2px solid #005A81;
}

QPushButton:pressed {
background: #0323FF;
border: 3px solid #0000FF;
}</string>
     </property>
     <property name="text">
      <string>Re-ajustar </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCantidadDatos">
     <property name="text">
      <string>Datos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelMedia">
     <property name="text">
      <string>Media:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDesvEst">
     <property name="text">
      <string>Desv Est</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelRangoInf">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QxtSpanSlider" name="horizontalSlider">
       <property name="styleSheet">
        <string notr="true">
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRangoSup">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAplicarRango">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: #5cb85c;
border: 2px solid #4cae4c;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: #47a447;
border: 2px solid #398439;
}

QPushButton:pressed {
border: 3px solid #398439;
}</string>
       </property>
       <property name="text">
        <string>Aplicar Rango</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonRestaurar">
     <property name="styleSheet">
      <string notr="true">QPushButton{
color : white;
background-color: #5cb85c;
border: 2px solid #4cae4c;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: #47a447;
border: 2px solid #398439;
}

QPushButton:pressed {
border: 3px solid #398439;
}</string>
     </property>
     <property name="text">
      <string>Restaurar Rango Completo Graficos</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header>qxtspanslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
